pkg/block/block_linux_test.go

```diff
@@ -251,3 +251,34 @@ func TestDiskPartLabel(t *testing.T) {
 		t.Fatalf("Got label %s, but expected %s label", label, util.UNKNOWN)
 	}
 }
+
+func TestDiskTypeUdev(t *testing.T) {
+	if _, ok := os.LookupEnv("GHW_TESTING_SKIP_BLOCK"); ok {
+		t.Skip("Skipping block tests.")
+	}
+	baseDir, _ := ioutil.TempDir("", "test")
+	defer os.RemoveAll(baseDir)
+	ctx := context.New()
+	ctx.Chroot = baseDir
+	paths := linuxpath.New(ctx)
+	fsType := "ext4"
+
+	_ = os.MkdirAll(paths.SysBlock, 0755)
+	_ = os.MkdirAll(paths.RunUdevData, 0755)
+
+	// Emulate a disk with one partition with label TEST_LABEL_GHW
+	_ = os.Mkdir(filepath.Join(paths.SysBlock, "sda"), 0755)
+	_ = os.Mkdir(filepath.Join(paths.SysBlock, "sda", "sda1"), 0755)
+	_ = ioutil.WriteFile(filepath.Join(paths.SysBlock, "sda", "sda1", "dev"), []byte("259:0\n"), 0644)
+	_ = ioutil.WriteFile(filepath.Join(paths.RunUdevData, "b259:0"), []byte(fmt.Sprintf("E:ID_FS_TYPE=%s\n", fsType)), 0644)
+	fs := diskPartTypeUdev(paths, "sda", "sda1")
```

Comment:
consider renaming `fsType` to `expectedPartType` and `fs` variables to `pt` to align with the "partition type" terminology (and yes, I realize udev uses "FS_TYPE" term :) )

Reply:
done!